monkestation/code/modules/reagents/chemistry/reagents/drink_reagents.dm

```diff
@@ -108,3 +108,24 @@
 		H.dna.features["mcolor"] = "fff"
 	H.regenerate_icons()
 	H.grant_language(/datum/language/sippins, TRUE, TRUE, "spray")
+
+
+
+/datum/reagent/consumable/ethanol/bureau
+	name = "Bureaucratic Perfection"
+	description = "Smells of freshly signed and filed papers.."
+	icons = "monkestation/icons/obj/drinks.dmi"  // created drinks.dmi
```

Comment:
"icon", not "icons"
Don't use double quotation marks, use single for files.

Reply:
ohh!!